main/src/main/java/org/mobilitydata/gtfsvalidator/validator/TripAndShapeDistanceValidator.java

```diff
@@ -98,15 +121,59 @@ static class TripDistanceExceedsShapeDistanceNotice extends ValidationNotice {
     /** The faulty record's shape max distance traveled. */
     private final double maxShapeDistanceTraveled;
 
+    /** The distance in meters between the shape and the stop. */
+    private final double distance;
+
     TripDistanceExceedsShapeDistanceNotice(
         String tripId,
         String shapeId,
         double maxTripDistanceTraveled,
-        double maxShapeDistanceTraveled) {
+        double maxShapeDistanceTraveled,
+        double distance) {
+      this.tripId = tripId;
+      this.shapeId = shapeId;
+      this.maxShapeDistanceTraveled = maxShapeDistanceTraveled;
+      this.maxTripDistanceTraveled = maxTripDistanceTraveled;
+      this.distance = distance;
+    }
+  }
+
+  /**
+   * The distance traveled by a trip should be less or equal to the max length of its shape.
+   *
+   * <p>The distance is less than the 11.1m threshold.
```

Comment:
Revised to clarify the threshold: `The distance between the last shape point and last stop point is less than the 11.1m threshold.`

Reply:
done ✅